docs/src/ensemble_kalman_sampler.md

```diff
@@ -1,21 +1,23 @@
 # Ensemble Kalman Sampling
 
 ### What Is It and What Does It Do?
-The Ensemble Kalman Sampler (EKS) ([Garbuno-Inigo et al, 2019](https://arxiv.org/pdf/1903.08866.pdf), [Cleary et al, 2020](https://clima.caltech.edu/files/2020/01/2001.03689.pdf), [Garbuno-Inigo et al, 2020](https://arxiv.org/pdf/1912.02859.pdf)) is a derivative-free method that can be used to solve the inverse problem of finding the optimal model parameters given noisy data. In contrast to Ensemble Kalman Inversion (EKI) ([Iglesias et al, 2013](http://dx.doi.org/10.1088/0266-5611/29/4/045001)), the EKS method approximately samples from the posterior distribution; that is, EKS provides both point estimation (through the mean of the final ensemble) and uncertainty quantification (through the covariance of the final ensemble), unlike EKI, which only provides the former.
-
+The Ensemble Kalman Sampler (EKS) ([Garbuno-Inigo et al, 2019](https://arxiv.org/pdf/1903.08866.pdf), [Cleary et al, 2020](https://clima.caltech.edu/files/2020/01/2001.03689.pdf), [Garbuno-Inigo et al, 2020](https://arxiv.org/pdf/1912.02859.pdf)) is a derivative-free tool for approximate Bayesian inference. It does so by approximately sampling from the posterior distribution. That is, EKS provides both point estimation (through the mean of the final ensemble) and uncertainty quantification (through the covariance of the final ensemble), this is in contrast to EKI, which only provides point estimation.
```

Comment:
I have a preference toward linking to the DOI, not the pdf directly. It makes it easier to access reference tools, etc, and it is the only link that is guaranteed to be maintained.

Reply:
good catch - I didn't spot that these werent DOIs